apps/explorer/src/components/longtext/Longtext.tsx

```diff
@@ -97,33 +96,25 @@ function Longtext({
     if (isLink) {
         if (category === 'unknown') {
             textComponent = (
-                <div className={styles.longtext} onClick={navigateUnknown}>
+                <Link variant="mono" onClick={navigateUnknown}>
                     {alttext ? alttext : text}
-                </div>
-            );
-        } else if (category === 'ethAddress') {
-            textComponent = (
-                <ExternalLink
-                    href={`https://etherscan.io/address/${text}`}
-                    label={text}
-                    className={styles.longtext}
-                />
+                </Link>
             );
         } else {
             textComponent = (
-                <div>
-                    <LinkWithQuery
-                        className={styles.longtext}
-                        to={`/${category}/${encodeURIComponent(text)}`}
-                    >
-                        {alttext ? alttext : text} {iconButton}
-                    </LinkWithQuery>
-                </div>
+                <Link
```

Comment:
I’m fine having this updated but also we really should just deprecate this component and replace it with updated UI component version.